.github/workflows/swap-azure-web-app-slots.yml

```diff
@@ -118,6 +145,80 @@ jobs:
             -WebAppName ${{ inputs.app-name }} `
             -SlotName ${{ inputs.destination-slot-name }}
 
+      - name: Checkout Repository
+        if: inputs.swap-code-repo != ''
+        uses: Lombiq/GitHub-Actions/.github/actions/checkout@dev
+        with:
+          repository: ${{ inputs.swap-code-repo }}
+          token: ${{ secrets.TAG_DEPLOYMENT_TOKEN }}
+
+      - name: Setup Git Config
+        if: inputs.swap-code-repo != ''
+        run: |
+            git config user.email '41898282+github-actions[bot]@users.noreply.github.com'
+            git config user.name 'github-actions[bot]'
+
+      - name: Fetch tags
+        if: inputs.swap-code-repo != ''
+        run: |
+            git fetch --tags
+
+      - name: Determine timestamp
+        if: inputs.swap-code-repo != ''
+        id: determine-timestamp
+        run: |
+            # Get the current timestamp in the format of yyyy-MM-dd-HH.mmUTC - can't use : in tags
+            "timestamp=$(Get-Date -Format 'yyyy-MM-dd-HH.mmUTC')" >> $Env:GITHUB_OUTPUT
+
+      - name: Delete Old prefix/latest tag
+        if: inputs.swap-code-repo != ''
+        run: |
+            $latestTag = (git tag -l '${{ inputs.tag-prefix }}/latest')
+            if ($latestTag)
+            {
+              git tag -d '${{ inputs.tag-prefix }}/latest'
+              git push origin ':refs/tags/${{ inputs.tag-prefix }}/latest'
+            }
```

Comment:
This can by DRYd with the other workflow, with a composite action (or a common script, but you can't easily use common scripts from workflows, as opposed to actions, what we also utilize with the `Scripts` folder).

Reply:
Did this but ended up splitting it into 2 steps. Removing the old tags and setting the timestamp are completely unrelated steps which could be a bit confusing - especially because it produces that output. Would also make for an interesting action name `remove-old-tags-and-set-timestamp`.